CodePush.js

```diff
@@ -70,7 +70,12 @@ async function checkForUpdate(deploymentKey = null) {
    *    because we want to avoid having to install diff updates against the binary's
    *    version, which we can't do yet on Android.
    */
-  if (!update || update.updateAppVersion || localPackage && (update.packageHash === localPackage.packageHash) || !localPackage && config.packageHash === update.packageHash) {
+  if (
+    !update || 
```

Comment:
What if you moved this expression to the same line as the `(`? That might make this look a little cleaner, and then align the other three with it.